lib/afterpay.rb

```diff
@@ -37,6 +37,8 @@ class << self
     # @!attribute server
     attr_writer :server
 
+    attr_accessor :user_agent
```

Comment:
Can you add a description to this attribute like the other attributes?

Reply:
Done!